company/okrs/2020_q1.md

```diff
@@ -0,0 +1,103 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. [$N][N] net new ARR
+   1. Sales leadership owns revenue targets and hiring plan
+   1. Sales, engineering, and product have a working process for prioritization and delivery
+1. **CEO: Aspirational, efficient all-remote team**
+   1. Roles and responsibilities are documented and maintained in handbook
+   1. Team members rely on the handbook for cross-functional info and processes
+   1. The handbook is fast to browse and search
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
```

Comment:
@juliasourceress These sales OKRs should reflect the sales plan you are working on. Should we consider these pending until the plan is ready?